weblab/static/js/run_experiment.js

```diff
@@ -0,0 +1,51 @@
+
+var RunExperiment = function() {};
+
+RunExperiment.prototype = {
+  init: function() {
+    $("#checkallbutton").click (function () {
+        $(".latestexperimentCheckBox").each (function () {
+            if (!$(this).prop('checked')) {
```

Comment:
I think you can test with `if ($(this).prop('checked'))` and set with `$(this).prop('checked', true);` - that might get it working?

Also, if you're just setting not toggling, it's probably not worth the `if` guard.

Reply:
Yeay it worked thanks you